test/rest.test.js

```diff
@@ -994,6 +994,37 @@ describe('strong-remoting-rest', function() {
         });
     });
 
+    it('should not flatten arrays for non-standard target type', function(done) {
+      var method = givenSharedStaticMethod(
+        function(arg, cb) { cb(null, { value: arg }); },
+        {
+          // Note: the string "array" is not a valid type
```

Comment:
For the purpose of the test, I can use `accepts: { arg: 'arg', type: 'any' }` if that's make the test easier to understand.